.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
```

Comment:
(B) it seems this addition came in late; tons of stuff got committed into `bin/`; can we have it purged? Best way is to rewrite the history with `bin/` removed and force-push to the same branch.

Reply:
I will try to do those changes as soon as I can